qemu/tests/cfg/secure_img.cfg

```diff
@@ -0,0 +1,16 @@
+- secure_img:
+    only s390x
+    type = secure_img
+    take_regular_screendumps = no
+    start_vm = yes
+    kill_vm_before_test = yes
+    kill_vm = yes
+    machine_type_extra_params = "usb=off,dump-guest-core=off,confidential-guest-support=lsec0"
+    extra_params = "-object s390-pv-guest,id=lsec0"
+    nic_extra_params = "devno=fe.0.0001"
+    virtio_serial_extra_params = "devno=fe.0.0002"
+    secure_params = 'echo "$(cat /proc/cmdline) swiotlb=262144" > /home/parmfile'
+    boot_img = 'genprotimg -k /home/HKD-8561-024B428.crt -p /home/parmfile -i %s -r %s -o /boot/secure-linux --no-verify'
```

Comment:
Add a note to let users know where the crt file is from, maybe the name can be as simple as `HKD.crt`?

Reply:
updated, please help me check if this is expected @PaulYuuu